docs/data/data-grid/aggregation/AggregationRemoveFunctionOneColumn.tsx

```diff
@@ -0,0 +1,47 @@
+import * as React from 'react';
+import { DataGridPremium, GridColDef } from '@mui/x-data-grid-premium';
+import { useMovieData } from '@mui/x-data-grid-generator';
+
+const COLUMNS: GridColDef[] = [
+  { field: 'title', headerName: 'Title', width: 200, groupable: false },
+  {
+    field: 'gross',
+    headerName: 'Gross',
+    type: 'number',
+    width: 150,
+    groupable: false,
+    valueFormatter: ({ value }) => {
+      if (!value || typeof value !== 'number') {
+        return value;
+      }
+      return `${value.toLocaleString()}$`;
+    },
+  },
+  {
+    field: 'year',
+    headerName: 'Year',
+    type: 'number',
+    availableAggregationFunctions: ['max', 'min'],
+  },
+];
+
+export default function AggregationRemoveFunctionOneColumn() {
```

Comment:
Shouldn't aggregation function name only be used in the aggregated column header?
<img width="874" alt="Screenshot 2022-06-06 at 11 21 11" src="https://user-images.githubusercontent.com/13808724/172134242-19fd6b45-b775-4796-8889-4f2c8bfbb4d1.png">

<img width="841" alt="Screenshot 2022-06-06 at 11 21 34" src="https://user-images.githubusercontent.com/13808724/172134302-65c27baf-2ff2-4272-9c87-8313c3283bd2.png">

Reply:
Oh good point...
I should probably override the `renderHeader` property instead of `headerName`

Reply:
Done